content/docs/guide-development.md

```diff
@@ -117,7 +117,51 @@ https://renkulab.io/
 
+To support the development of workflows in Renku, a set of dedicated funcitonailities, provided as Renku plugins, are made available. In particular those aim to:
 
+* Offer an interactive (and non) visualization of the project Knowledge Graph (`renku-graph-vis` plugin)
+* Create dedicated annotations for the execution of `astroquery` functions and add those to the project Knowledge Graph (`renku-aqs-annotation` plugin)
+
+Those two set of features are provided within different plugins.
+
+### Visualizing project Knowledge Graph with `renku-graph-vis` plugin
+
+Starting from the knowledge graph, this is used to generate an interactive graphical representation of the graph itself.
+
+The plugin provides two CLI commands:
+  * `display` to generate a representation of the graph over an output image
+  * `show-graph` to start an interactive visualization of the graph over the browser
+
+In addition, the plugin offers a dynamic visualization of the graph during an interactive renku session.
+
+The primary benefit introduced is the ability to have a live overview of the ongoing development, during an interactive Renku session. This can be seen within the animation below, where the graph is automatically updated with information about the execution of a notebook upon its completion.
+
+![](renkulab_execution_example_2.gif)
+
+Additionally, an ontology can be integrated into the graph, providing valuable insights into the types of entities inside it. In the image below, it can be seen that the `SimbadClass` node is an instance of the `AstroqueryModule` class, while `Mrk 421` is an instance of the `AstrophysicalObject` class.
+
+![](details_astroquery_annotations_2.png)
+
+More technical details are presented in the README of the repo page: [https://github.com/oda-hub/renku-graph-vis/](https://github.com/oda-hub/renku-graph-vis/)
+
+### Tracking access to astronomical archives and services in the project Knowledge Graph by using `renku-aqs-annotation` plugin
+
+By this plugin, the calls amongst the most important `astroquery` methods are intercepted and specific annotations are generated. Those will be then added to the project Knowledge Graph: [https://github.com/oda-hub/renku-aqs-annotation](https://github.com/oda-hub/renku-aqs-annotation)
+
+In the image below, the information added to the project Knowledge Graph is highlighted. Specifically, it can be seen that during a papermill run of the `test-notebook.ipynb` notebook (that produced `out.ipynb` as an output notebook) a call to the astroquery method `query_object`, via the `Simbadclass`, has been detected. This, is requesting the object `Mrk 421`. The hightlighed labels on the edges provide information about the relationship between the two nodes: during the `papermill` execution, a call to the query_object method is executed (`call` label) and in turn, this requests the Astrophysical Object `Mrk 421`.
+
+![](details_astroquery_annotations_1.png)
+
+### Installation
```

Comment:
installation section is at the same level as other sections of the readme which is not right since it's not clear to what does it refer.

instead installation commands could be part of the section of each plugin, at the end of each section perhaps just a short sentence and a code snipplet.

Reply:
they're now in the relative sections, with a more concise explanation